.github/workflows/docker-push.yml

```diff
@@ -33,19 +49,12 @@ jobs:
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
 
-      - name: Build and push Docker image
+      - name: Build and push docker image
         uses: docker/build-push-action@v6.1.0
         with:
           context: .
+          file: e2e.Dockerfile
```

Comment:
Can we rename e2e.Dockerfile to Dockerfile so it's not confusing?

We will be using that file for e2e and providing docker images.